### DISCUSSION TITLE
Is there a way to tell if a document exists on the RECAP archive without permissions?

### DISCUSSION THREAD

**Discussion by pcd444** [freelawproject/courtlistener#4177] [Q&A]
Looking through the docs the only place that I found where it tells you if the document is on pacer or not is the recap documents api (/api/rest/v3/recap-documents/) where it has the is_available field. But this endpoint requires permissions.

Is there a way to find this out without permissions?

**Comment by mlissner** [comment]
You could probably use the search API, but it's not always the best for this kind of thing. What do you have in mind?
